client/src/components/helpers/Wrapper.js

```diff
@@ -1,6 +1,7 @@
 import styled from "styled-components/macro";
 
 export const ContentWrapper = styled.div`
+  position: relative;
```

Comment:
I think when you use position relative, you need to use position: absolute in the container around..